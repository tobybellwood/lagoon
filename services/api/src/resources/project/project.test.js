const { Sql } = require('./sql');

describe('Resolvers', () => {
  describe('updateProject', () => {
    it('should return proper update statement', () => {
      const input = {
        id: 1,
        patch: {
          name: 'test',
        },
      };
      const ret = Sql.updateProject(input);

      expect(ret).toMatchSnapshot();
    });
  });

  describe('selectProject', () => {
    it('should create query', () => {
      const ret = Sql.selectProject(1);
      expect(ret).toMatchSnapshot();
    });
  });
});
