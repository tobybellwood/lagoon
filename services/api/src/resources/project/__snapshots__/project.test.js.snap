// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resolvers selectProject should create query 1`] = `"select * from \`project\` where \`id\` = 1"`;

exports[`Resolvers updateProject should return proper update statement 1`] = `"update \`project\` set \`name\` = 'test' where \`id\` = 1"`;
